communication/README.md

```diff
@@ -76,7 +76,7 @@ We talk a lot, too.
 
 ### Slack
 
-Our real-time platform with Kubernetes enthusiasts spread across 250+ channels.
+Our real-time platform with Kubernetes enthusiasts is spread across 250+ channels.
```

Comment:
the problem with this line is that it contains two sentence fragments.  If you're going to improve them, you should make them complete sentences.